references/intro/diff/Loop3-Loop4.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Now I [see](http://htmlpreview.github.io/?https://github.com/SingaporeScalaProgrammers/scala-workshop/blob/f70c2f7f26f35fcc16c7e3cc6bf3c34340fba6f0/references/intro/diff/Loop3-Loop4.html) what did you mean with the diff.